packages/components/psammead-image-placeholder/src/index.amp.jsx

```diff
@@ -0,0 +1,91 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+import { bool } from 'prop-types';
+import { C_LUNAR, C_SHADOW } from '@bbc/psammead-styles/colours';
+import { BBC_BLOCKS, BBC_BLOCKS_DARK_MODE } from '@bbc/psammead-assets/svgs';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+const bgImageDark = `data:image/svg+xml;base64,${BBC_BLOCKS_DARK_MODE}`;
+const bgImageRegular = `data:image/svg+xml;base64,${BBC_BLOCKS}`;
+
+const AmpImgPlaceholderContainer = ({
```

Comment:
Attempted to use emotion to create a styled component instead of using the inline style method. However currently `fallback` isn't supported in the installed version of emotion as a valid prop so it is being stripped out on render. This has been fixed here: https://github.com/emotion-js/emotion/issues/2224 and is now supported in the latest version of emotion.

A seperate issue will be created to amend this once all the emotion issues are fixed within Simorgh and the psammead is updated to the latest version of emotion.

This is what it should look like:

```jsx
const AmpImgPlaceholderContainer = styled.div`
  background-color: ${({ darkMode }) => (darkMode ? C_SHADOW : C_LUNAR)};
`;
```

Reply:
We have upgraded to Emotion 11, have we not got the version with this fix?

Reply:
We're currently on version `11.0.0` https://github.com/bbc/psammead/blob/4737f88e2cf5efab353b32c916817c2615d23c23/package.json#L46-L48

There was a new version `11.1.5` that came out a few days ago which included the fix from https://github.com/emotion-js/emotion/issues/2224.

But it looks like it's been reverted https://www.npmjs.com/package/emotion, probably due to the Opera mini mishap. So currently there isn't a live build with the fix. Although I tested it before it was reverted and it worked a treat👌

Reply:
We've put it back to a pretty new version, I think we might wanna upgrade and use the new feature.

Reply:
Yep definitely agree, going to merge this PR in first and apply the improvements in a follow up PR as this has been ongoing for quite some time now.